Dockerfile

```diff
@@ -6,4 +6,5 @@ RUN mkdir -p /home/nginx/.cloudvolume/secrets && chown -R nginx /home/nginx && u
 COPY . /app
 RUN pip install pip==20.0.1 \
     && pip install --no-cache-dir --upgrade -r requirements.txt \
-    && pip install --upgrade git+https://github.com/seung-lab/PyChunkedGraph.git@master
\ No newline at end of file
+    && pip install --upgrade git+https://github.com/seung-lab/PyChunkedGraph.git@master \
```

Comment:
Why is the PyChunkedGraph required?

Reply:
for ingest, but we could maintain two docker files, it is not required to run frontend